src/open_inwoner/openklant/wrap.py

```diff
@@ -31,21 +25,27 @@ def fetch_klantcontactmomenten(
     if not user_bsn and not user_kvk_or_rsin:
         return []
 
+    client = build_client("klanten")
+    if client is None:
+        return []
```

Comment:
I think it would be better to raise an error in `build_client` if no service was found, that way we can properly handle the situation in which configuration is missing, but that might be tackled in a separate issue

Reply:
Yes, but we should also handle connection and response errors so I'd make it a new ticket.